CHANGELOG/CHANGELOG-1.20.md

```diff
@@ -2388,4 +2388,4 @@ filename | sha512 hash
 - github.com/godbus/dbus: [ade71ed](https://github.com/godbus/dbus/tree/ade71ed)
 - github.com/xlab/handysort: [fb3537e](https://github.com/xlab/handysort/tree/fb3537e)
 - sigs.k8s.io/structured-merge-diff/v3: v3.0.0
-- vbom.ml/util: db5cfe1
\ No newline at end of file
+- vbom.ml/util: db5cfe1
```

Comment:
Is this diff here intentional?

Reply:
sorry for that. As I edit the file on phone.

Reply:
@hasheddan fixed now.